department-of-veterans-affairs/caseflow
# Kamalam7/appeals 29197

Resolves (https://jira.devops.va.gov/browse/APPEALS-29197)
# Description
Created a new Review Package page. The Mail team user can access this page. Created components for heading and buttons for Cancel, Intake appeal and Create record.
 
## Acceptance Criteria
- [x] A Mail Team User can access a new page called "Review Package"
the heading of this page is "Review Package"
A mail Team user can click a link on the bottom of the page titled "cancel"
This will return the user to the Correspondence Queue
This will discard any unsaved changes
A mail team user is able to click a button titled "Create Record"
This will direct the user to the Intake Page
If the feature flag :correspondence_queue is disabled, then the user is routed to /unauthorized

## Testing Plan

1. Login as a Mail team user ([JOLLY_POSTMAN]
2. From the App selector, select and click on Queue
3. In Your cases page, click on Switch views and select Correspondance Cases
<img width="886" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/63c0bb21-8660-4b4e-bfe2-4c2bec29d569">
4. From the Correspondance Cases, click on Veteran name or ID, then it will go to Review Package page 
<img width="883" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/2b8e2b67-00a8-48d7-a3ff-5acac6650b3b">
5. In the Review package page below you can see Cancel button, when you click on Cancel it will route to queue/correspondance page
6. When you click on Create record button, it will route to Intake page





1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
